inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/fused_conv_eltwise/fused_conv_eltwise_kernel_b_fs_yx_fsv4.cpp

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include "fused_conv_eltwise_kernel_b_fs_yx_fsv4.h"
+#include <vector>
+#include <utility>
+#include <algorithm>
+
+namespace kernel_selector {
+constexpr size_t sub_group_size = 16;
+
+fused_conv_eltwise_kernel_b_fs_yx_fsv4::fused_conv_eltwise_kernel_b_fs_yx_fsv4()
+    : fused_conv_eltwise_kernel_base("fused_conv_eltwise_gpu_b_fs_yx_fsv4") {
+}
+
+ParamsKey fused_conv_eltwise_kernel_b_fs_yx_fsv4::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
```

Comment:
Same concern on supported data types as in the other kernel

Reply:
Fused_conv_eltwise kernel removed from this PR.